bot.user.js

```diff
@@ -1038,19 +1074,35 @@ var userInterface = window.userInterface = (function () {
                 if (e.keyCode === 71) {
                     userInterface.toggleLeaderboard(!window.leaderboard);
                 }
+                // Letter 'H' to toggle hidden mode
+                if (e.keyCode === 72) {
+                    userInterface.toggleOverlays();
+                }
                 // Letter 'I' to toggle autorespawn
                 if (e.keyCode === 73) {
                     window.autoRespawn = !window.autoRespawn;
                     window.log('Automatic Respawning set to: ' + window.autoRespawn);
                     userInterface.savePreference('autoRespawn', window.autoRespawn);
                 }
-                // Letter 'H' to toggle hidden mode
-                if (e.keyCode === 72) {
-                    userInterface.toggleOverlays();
+                // Letter 'W' to auto rotate skin
```

Comment:
Apparently they are client side as I have never seen anyone with rotating skin and IMO it is just extra client side code, putting extra burden on the CPU...